docs/encyclopedia/temporal-nexus-overview.mdx

```diff
@@ -0,0 +1,118 @@
+Synchronous Nexus `Operations` can also execute arbitrary code, but unlike `Activities` they have a short request timeout (under 10 seconds) and don’t support [heartbeats](https://community.temporal.io/t/best-practices-for-long-running-activities/934).
+
+Nexus RPC is designed for arbitrary-duration operations whose lifetime may extend beyond a traditional RPC and is an underpinning to connect durable executions within and across Namespaces, clusters and regions – with a clean API contract to streamline multi-team collaboration. Any service can be exposed via a Nexus endpoint as a set of sync or async Nexus operations – the latter provides an operation identity and a uniform interface to get the status of an operation or its result, receive a completion callback, or cancel the operation.
+
+Nexus Services and Operations are often registered in the same Worker as the underlying Temporal primitives they abstract, and use the same task-queue worker polling architecture as Workflows and Activities.
+
+To make a Nexus Service available for others to use, a Nexus Endpoint must be created in the Nexus API Registry with a target Namespace and task-queue. 
+In pre-release, all Nexus requests will be sent to the single target Namespace and task-queue for the Endpoint. Future releases may support multiple Endpoint targets, similar to how an API proxy can have multiple targets.
+
+When you start a Nexus Operation in a caller Workflow, a command is sent to Temporal to schedule the Operation, and then the Temporal Nexus Machinery is responsible for making the Nexus RPC calls on your behalf. 
+This means you don’t have to use Nexus RPC directly, only the Temporal SDK along with a Temporal Service.
+
+Similar to executing an Activity, for a Nexus Operation:
+
+1. Caller Workflow will schedule a Nexus Operation using the Temporal SDK, which sends a `ScheduleNexusOperation` command in the existing `RespondWorkflowTaskCompleted` RPC which is recorded as `NexusOperationScheduled` in the caller’s Workflow history.
+2. Nexus handler Worker uses the Temporal SDK to `PollNexusTaskQueue` and process the Nexus request.
+3. Nexus handler Worker sends a `RespondNexusTaskCompleted` via the SDK, with the result of the Nexus request, which may be one of: Started, Completed, Failed, TimedOut, Canceled,  see Nexus Operation Events for more info
+4. Caller Workflow then processes the Nexus Operation event via `PollWorkflowTaskQueue`.
+
+See also: [Nexus Architecture on GitHub](https://github.com/temporalio/temporal/blob/main/docs/architecture/nexus.md).
+
+## Key Capabilities
+
+Nexus support is available in Temporal Cloud, OSS Server, and Temporal SDKs:
+
+- Workers that handle Nexus Operations, listening on a Namespace and Task Queue
+- Nexus API registry, to create Nexus Endpoints that forward requests to a target Namespace and Task Queue, so others can use Nexus services via a Nexus endpoint.
```

Comment:
```suggestion
- Nexus API registry, to create Nexus Endpoints that forward requests to a target Namespace and Task Queue, so others can use Nexus services via a Nexus endpoint
```

Reply:
Replace "forwards" with "routes"

Reply:
The overview has been replaced via internal doc.